app/src/main/java/org/systers/mentorship/view/activities/SignUpActivity.kt

```diff
@@ -100,8 +102,8 @@ class SignUpActivity : BaseActivity() {
             tiEmail.error = null
         }
 
-        if (password.isBlank()) {
-            tiPassword.error = getString(R.string.error_empty_password)
+        if (password.length < 8 || password.length > 25) {
```

Comment:
Can these integers be in resources file, somewhere that can be separate and have a clear definition.
Does this change makes sense? What do you think?

In this way this can be also used as parameters to the `error_wrong_password_length ` string, instead of then if we have to change these values, having to change both here and in the string.

Reply:
@isabelcosta Thank you for a review! You are absolutely right! I added two constants: `PASSWORD_MIN_LENGTH` and `PASSWORD_MAX_LENGTH` and updated the String :)